Explain what the `Hush-Hush (Expert)` category means in `Medals` (#10112)
ppy/osu-wiki: wiki/Medals/en.md

As the Hush-Hush medals are meant to be secret achievements of some sort, the unlock requirements are not directly specified; however, players are [welcome to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).

> Hush-Hush also has its sub-category called "Hush-Hush (Expert)" the difference of which is medals in it don't allow any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods usage.

Reviewer: ```suggestion
Hush-Hush also has its sub-category called "Hush-Hush (Expert)". The difference is that medals in it do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.
```
